doc-content/enterprise-only/installation/jws-zip-install-proc.adoc

```diff
@@ -1,47 +1,69 @@
 [id='jws-zip-install-proc']
 
-= Installing {KIE_SERVER} with a ZIP file
+= Installing {KIE_SERVER} from ZIP files
 
-{KIE_SERVER} provides the runtime environment for business assets and accesses the data stored in the assets repository (knowledge store). You can use the `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip` file to install {KIE_SERVER} on an existing {JWS} {JWS_VERSION_LONG} or higher server instance.
+{KIE_SERVER} provides the runtime environment for business assets and accesses the data stored in the assets repository (knowledge store). You can use zip files to install {KIE_SERVER} on an existing {JWS} {JWS_VERSION_LONG} or higher server instance.
```

Comment:
"ZIP" - Should be capped per our style. Search all. You go back and forth a bit I think. There's actually more to this story that I spoke with Ingrid about back when I first started here. It should be ZIP, but I think there's some kind of proprietary thing with that name such that using it this way is a violation of some kind. Not a matter for this PR as much as a global one that I was just reminded of. Will see what we do.